lib/task-metrics/index.js

```diff
@@ -30,7 +30,7 @@ module.exports = settings => {
     const query = { start, end };
 
     logger.debug('fetching internal-deadlines report');
-    const internalDeadlinesData = await metrics('/reports/internal-deadlines', { stream: false, query }, accessToken);
+    const internalDeadlinesData = (await metrics('/reports/internal-deadlines', { stream: false, query }, accessToken)).filter(Boolean);
```

Comment:
What's the need for the `.filter`? And if it _is_ needed, can it move down two lines to just before the `.forEach` to be a bit neater?

Reply:
The report's `parse()` method returns `null` if a task doesn't exceed the deadline:
https://github.com/UKHomeOffice/asl-metrics/blob/master/lib/reports/internal-deadlines/index.js#L103

Maybe it would be better to filter them at source here: https://github.com/UKHomeOffice/asl-metrics/blob/master/lib/reports/index.js#L33-L39

Reply:
I agree that it would be better to filter them at source, but it's probably ok to also filter here* too.

\* actually just down there 👇

Reply:
```suggestion
    const internalDeadlinesData = await metrics('/reports/internal-deadlines', { stream: false, query }, accessToken);
```